CommonTools/CandUtils/interface/makeCompositeCandidate.h

```diff
@@ -18,7 +18,7 @@ namespace helpers {
   private:
     std::unique_ptr<reco::CompositeCandidate> cmp_;
     std::unique_ptr<reco::Candidate> release() {
-      std::unique_ptr<reco::Candidate> ret(std::move(cmp_.get()));

```

Comment:
I don’t think either of these is correct in this case. It looks like this was supposed to be
```C++
return std::unique_ptr<reco::Candidate>(std::move(cmp_));
```

Reply:
Agreed.

`clang-tidy` doesn't try to correct buggy code, only to preserve the existing behaviour :-)

Reply:
IIUC just
```cpp
return std::move(cmp_);
```
could be enough.

Reply:
I had wondered if that would work or not.